src/app/pages/ParatimeDashboardPage/ParaTimeSnapshot.tsx

```diff
@@ -54,8 +56,8 @@ export const ParaTimeSnapshot: FC<{ scope: SearchScope }> = ({ scope }) => {
           <Nodes scope={scope} />
         </StyledGrid>
         <StyledGrid item xs={22} md={5}>
-          {scope.network === Network.mainnet && <RosePriceCard />}
-          {scope.network === Network.testnet && <TestnetFaucet layer={scope.layer} />}
+          {ticker === Ticker.ROSE && <RosePriceCard />}
+          <TestnetFaucet network={scope.network} layer={scope.layer} ticker={ticker} />
```

Comment:
looks a bit odd without the condition; maybe `{link && <TestnetFaucet link={link} />}`

Reply:
done